dali/operators/generic/erase/erase.cc

```diff
@@ -90,8 +93,11 @@ coordinates. Since no axis_names is provided, the anchor and shape must contain
 except "C" (channels).
 What gives::
 
-    output[y, x, c] = 0               if 0.15 * 300 <= x < (0.15 * 300) + 20 and (0.15 * 300) <= y < (0.15 * 300) + 30
-    output[y, x, c] = input[y, x, c]  otherwise
+    if 0.15 * 300 <= x < (0.15 * 300) + 20 and
+       (0.15 * 300) <= y < (0.15 * 300) + 30:
+      output[y, x, c] = 0
+    else:
+      output[y, x, c] = input[y, x, c]  otherwise
```

Comment:
If you make it look _that_ much like Python, then it better be Python.
```suggestion
    if (0.15 * 300 <= x < (0.15 * 300) + 20 and
       (0.15 * 300) <= y < (0.15 * 300) + 30):
      output[y, x, c] = 0
    else:
      output[y, x, c] = input[y, x, c]
```

Reply:
Done